### PR TITLE
Consumable adapter: add cats and adomain to bids

### PR DESCRIPTION
**Pull request by jpiros**
This PR adds support for the bid meta object in the bid response

### CODE REVIEW

**Comment by guscarreon** [comment]
Good day @jpiros apparently your pull request is failing the `gofmt` part of our checks:
```
Run ./validate.sh --nofmt --cov --race 10
gofmt needs to be run, 13 files have issues.  Below is a list of files to review:
analytics/config/config.go
analytics/core.go
analytics/filesystem/file_module.go
currency/rates.go
endpoints/openrtb2/video_auction.go
exchange/exchange_test.go
firstpartydata/first_party_data.go
metrics/config/metrics.go
router/router.go
stored_requests/backends/http_fetcher/fetcher.go
stored_requests/events/http/http.go
util/jsonutil/jsonutil.go
version/version.go
Error: Process completed with exit code 1.
```

To make this go away you can run `gofmt` on the entire package doing:
```
go fmt .
```

**Review by hhhjort — DISMISSED** [review]
LGTM

**Review by hhhjort — DISMISSED** [review]

File: adapters/consumable/consumable.go [hunk 1]
```diff
@@ -77,6 +77,10 @@ type decision struct {
 	ImpressionUrl *string    `json:"impressionUrl,omitempty"`
 	Width         uint64     `json:"width,omitempty"`  // Consumable extension, not defined by Adzerk
 	Height        uint64     `json:"height,omitempty"` // Consumable extension, not defined by Adzerk
+	Adomain       []string   `json:"adomain,omitempty"`
+	Cats          []string   `json:"cats,omitempty"`
+	NetworkId     int        `json:"networkId,omitempty"`
```
Comment (mansinahar) [inline adapters/consumable/consumable.go:82]: I don't see the `NetworkId` and `MediaType` value being used anywhere. Are they still needed?

Comment (jpiros) [reply adapters/consumable/consumable.go]: Missed out on removing those. Should be updated now to reflect changes.

Comment (mansinahar) [reply adapters/consumable/consumable.go]: Perfect. Thanks @jpiros!

**Review by mansinahar — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]

**Pull request merged**